public/locales/en/files.json

```diff
@@ -37,6 +37,12 @@
     "descriptionFolder": "{count, plural, one {Are you sure you want to delete this folder? This action is permanent and cannot be reversed.} other {Are you sure you want to delete these {count} folders? This action is permanent and cannot be reversed.}}",
     "descriptionFile": "{count, plural, one {Are you sure you want to delete this file? This action is permanent and cannot be reversed.} other {Are you sure you want to delete these {count} files? This action is permanent and cannot be reversed.}}"
   },
+  "pinningModal": {
+    "title": "Select the service(s) with which you wish to pin these items.",
```

Comment:
Seeking thoughts from @rafaelramalho19 and @lidel here ... It's a little weird to not offer any explanatory context for "why do I only have one choice and what's a service??" in this modal:
![image](https://user-images.githubusercontent.com/1507828/98158335-ef61ef80-1e97-11eb-867c-d33e0137a516.png)
That said, I think we're probably OK to leave as-is if this is only (a) a short-term situation and (b) only visible to early adopters loading master from webui.ipfs.io. Your thoughts?

Reply:
We can even have a release with this. It just transitions the users to the new flow, and in the patch notes they can keep track of what's being changed too.

Don't think it's necessary to specify why we only have a service for now, I even think it smoothens the transition between local pinning only into remote pinning 😄